plancklens/filt/filt_cinv.py

```diff
@@ -327,7 +327,7 @@ def _calc_mask(self):
 
     def _ninv_hash(self):
         ret = []
-        for ninv_comp in self.ninv[0]:
+        for ninv_comp in self.ninv:
             if isinstance(ninv_comp, np.ndarray) and ninv_comp.size > 1:
```

Comment:
ninv from opfilt_pp.alm_filter_ninv([ninv_desc], t_transf, marge_qmaps=(), marge_umaps=()).get_ninv() is a **list of np.array**. This routine checks for list of np.**nd**array, causing plancklens to produce huge hashfiles

Reply:
Although this PR fixes initial problem, I propose to change line 330 instead and let it check for both, `np.ndarray`, and `np.array`